### DISCUSSION TITLE
Hasn't been working on any event pages since last redesign in Summer '21

### DISCUSSION THREAD

**Discussion by HuckToohey** [akaessens/NoFbEventScraper#38] [General]
Curious whether anyone has recently found an event page which parses properly to calendar format.

**Comment by akaessens** [comment]
> Curious whether anyone has recently found an event page which parses properly to calendar format.

Hi,
For me it's still working...

Can you provide an example event which doesn't work for you? 
Also, which android version and does it work on older releases?
